src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp

```diff
@@ -310,6 +310,87 @@ AttestationVerificationResult MapError(CertificateChainValidationResult certific
         return AttestationVerificationResult::kInternalError;
     }
 }
+
+// CertificateType class doesn't work since it doesn't encode PAA>
+enum AttestationChainElement : uint8_t
```

Comment:
can we do `enum class` on these enums to avoid auto-casting and enforce type safety?